SwiftGit2.xcodeproj/project.pbxproj

```diff
@@ -909,6 +917,10 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = BEB31F4E1A0D75EE00F525B9 /* iOS-Application.xcconfig */;
 			buildSettings = {
+				HEADER_SEARCH_PATHS = (
+					"$(OBJROOT)/UninstalledProducts/include",
```

Comment:
What is `UninstalledProducts`?

Reply:
I don't know I guess Xcode added it when I was changing the Swift version in the .xcodeproj file?